packages/@okta/vuepress-theme-prose/components/Sidebar.vue

```diff
@@ -104,23 +104,33 @@ export default {
         queue.push(nav);
         let current = queue.pop();
         while (current) {
-          if (current && current.subLinks) {
+          if (current?.subLinks) {
             queue.push(...current.subLinks);
-          } else if (current && current.guideName) {
+          } else if (current?.guideName) {
             // add sections
             current.subLinks = [];
             const guide = guidesInfo.byName[current.guideName];
-            if (guide && guide.sections) {
-              guide.sections.forEach(section => {
-                current.subLinks.push({
-                  title: section.title,
-                  path: section.makeLink(
-                    guide.frameworks.includes(framework)
-                      ? framework
-                      : guide.mainFramework
-                  )
+
+            if (Array.isArray(guide?.sections)) {
+              const [firstSection] = guide.sections;
+
+              // Special value for guide that only has one section and should be
+              // linked at the parent
+              if (firstSection.name === 'main') {
```

Comment:
Is it a pre-req for new single page guides to then have a section named "main"? Should we also check if the number of sections === 1?

Reply:
> Is it a pre-req for new single page guides to then have a section named "main"?
For now, yes. The docs teams is OK with a convention of always naming it `main`. 

> Should we also check if the number of sections === 1?
I was thinking about only using the heuristic of `if sections === 1` to always denote a single page guide. 

I'll add this check too.

If down the line we want to remove the naming convention and only use the `sections === 1` heuristic then it's simple to do so and allow names other than `main.

Reply:
Wouldn't a value like `single-page` be a bit more explicit down the road?

Reply:
> Wouldn't a value like single-page be a bit more explicit down the road?

Part of the issue is that the section name appears in the URL of the page, so we're trying to keep is semi-innocuous